### ISSUE TITLE
Cannot Build With Jaeger Exporter Due To Thrift Vanity

### ISSUE THREAD

**Issue by marwan-at-work** [census-instrumentation/opencensus-go#993]
**Describe the bug**
It seems that one of Jaeger's import paths use vanity imports and the live server is not working as expected: `curl -sSL git.apache.org/thrift.git/lib/go/thrift?go-get=1` returns a 302 to https://github.com/apache/thrift.git/lib/go/thrift?go-get=1 which then returns a 404. 

**To Reproduce**
1. Make sure you don't have thrift in your local cache (vendor, gopath, or pkg/mod) 
2. Build a new project with a Jaeger exporter

PS. The thrift repo on Github does not have an issue tracker...so I reported here :)

**Comment by zerok** [comment]
I'm currently running into a possibly related issue when using an Athens server as GOPROXY:

```
go: verifying git.apache.org/thrift.git@v0.0.0-20181218151757-9b75e4fe745a: checksum mismatch
	downloaded: h1:ROdxmPITVHDxX22/qs2j1XXyJ/riQh8MB4UeWA4ZfVw=
	go.sum:     h1:GnKpWvJXysBGdcWMEibN//UeRRJeLDUgsXjKXjagFLg=
```

Would it perhaps make sense to depend directly on the Github URL inside go.mod (e.g. using something like `replace` definition)?

**Comment by jbub** [comment]
Yes i am currently not able to use `opencensus-go` with Go modules because of this.

**Comment by odeke-em** [comment]
@zerok @marwan-at-work what versions of Go are y'all using? If using Go1.11.2 and earlier, please upgrade to Go1.11.4.

**Comment by jbub** [comment]
I am on `go version go1.12 darwin/amd64`

**Comment by jbub** [comment]
I made a reprocuder repo https://github.com/jbub/censusthrift and here is the failing CI build https://cloud.drone.io/jbub/censusthrift/1/1/2.

**Comment by marwan-at-work** [comment]
My original issue is not about Checksum Mismatches, it's about the fact that the vanity server is not set up correctly. The issue is still reproducible if you follow my original description.

**Comment by jbub** [comment]
Okay so i will open another issue as this seems like different issues.

**Comment by rghetia** [comment]
@jbub with which version of go was the entry for [thrift](git.apache.org/thrift.git v0.12.0 h1:692K1/SsOcQvkvMRMdt60FCq2AvKpuQNM6sIeH3mN4s=) created? If you remove the entry and recreate it with g01.12 it should work.

**Comment by jbub** [comment]
I thought i did it with Go 1.12 but apparently i did not ? This commit fixed it https://github.com/jbub/censusthrift/commit/d4bcd45d1081891bf2f8764f1e12b30bad42eef6.

**Comment by jbub** [comment]
Thanks for the help @rghetia.

**Comment by jbub** [comment]
But then again seems like a bug related to Athens, it works with GOPROXY="", but does not work with GOPROXY pointed to athens proxy.

**Comment by jbub** [comment]
Can confirm that the athens v0.3.0 fixes the issue, you just have to delete the old downloaded package from athens storage and then it will work.

**Comment by zerok** [comment]
Can confirm that 🙂

**Comment by rghetia** [comment]
fixed by #1052
